docs/applying-axioms.md

```diff
@@ -0,0 +1,48 @@
+Algorithm for applying one rule to a pattern
+============================================
+
+**Input:** `φ(X)`, `∀ Y .α(Y) -> ◍ β(Y)`
+
+Note: ◍ is strong next in this document
```

Comment:
Why that character? I have never seen a circle full of vertical stripes for that. Unicode has white and black circles in a few sizes: `⚪⚫○●`. Those are `MEDIUM? (BLACK|WHITE) CIRCLE`. (Maybe you looked only for character names starting with CIRCLE).